### ISSUE TITLE
Changing tab in Settings should make the scroll bar reset to top

### ISSUE THREAD

**Issue by junpet** [WDaan/VueTorrent#322]
Current behavior:
1. Open the Settings, we are on the `Vuetorrent` tab. Scroll to down. 
![image](https://user-images.githubusercontent.com/3465278/143184291-1855771b-7be7-4f4d-b912-abdb09eb10a2.png)
2. Click on one of the following tabs: `Downloads`, `Bittorrent`, `Webui`. See how the scroll bar won't reset to the top, but remains on the same position.
![image](https://user-images.githubusercontent.com/3465278/143184465-58ab166a-2f1f-4ba6-a40e-a24b0627a722.png)

Expected behavior:
I think the scroll bar should reset between changing tabs.

**Comment by WDaan** [comment]
You're absolutely right & it is annoying, but unfortunately I haven't found a way to fix this yet 😕
